### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by Farida-Hamid**
> This is the submission for the first page of the mobile app version of my personal portfolio.

### Tasks completed:
- Add the logo.
- Add a home icon.
- Add text introduction.
- Add icons of social media for links.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header>
+      <a id="logo">Farida's Logo</a>
+      <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
+    </header>
+    <section id="intro">
+      <h2>I'm Farida Glad to see you!</h2>
```
Comment (gemmen29) [inline index.html:16]: - [ ] Kindly change the `h2` tag to the `h1` tag since this is the main heading of the page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header>
+      <a id="logo">Farida's Logo</a>
+      <img src="./images/Union.jpg" alt="menue bar" id="menue-button">
+    </header>
```
Comment (gemmen29) [inline index.html:14]: - [ ] Kindly decrease the top spacing between the header and the top of the page to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        don’t hestiate to contact me.</p>
+      <h3>let's connect</h3>
+      <section id="social">
+        <img src="./images/Vector.png" alt="twitter">
+        <img src="./images/Linkedin-icon.jpg" alt="linkedin">
+        <img src="./images/Vector.jpg" alt="email">
+        <img src="./images/Vector-2.jpg" alt="github">
+        <img src="./images/Vector-3.jpg" alt="angel-list">
+      </section>
+    </section>
```
Comment (gemmen29) [inline index.html:29]: - [ ] Kindly increase the spacing at the top of this section to match the Figma design.

| Your design | Figma design |
| --- | --- |
| image | image |

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Farida-Hamid, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+#logo {
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+}
+
+#intro {
+  background: url("./images/header-shapes-mobile.jpg") no-repeat;
```
Comment (AnselemOdims) [inline style.css:25]: - [ ] [OPTIONAL] At the moment, the background image is not properly positioned to match the design. Please adjust it so it looks like the original design

**Review by AnselemOdims — APPROVED** [review]
### Hi  @Farida-Hamid 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**